nexus-repository-apk-it/pom.xml

```diff
@@ -72,6 +72,11 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
this one dependency also could be removed. I think it is not necessary to add it. Pretty sure if you remove it mvn clean install will pass.

Reply:
Indeed. nexus-testsupport dep is retrieve through nexus-repository-apk dep. I remove it too in commit dc7d34e8116468718f49b98fad330f566810ea5f